src/components/ResourceCard/AddResource/ResourceSelectActivity.js

```diff
@@ -69,16 +68,14 @@ let ResourceSelectActivity = (props) => {
             <div
               className="activity-img"
               style={{
-                backgroundImage: `url(${global.config.laravelAPIUrl}${activity.image})`,
+                backgroundImage: `url(${global.config.resourceUrl}${activity.image})`,
```

Comment:
It will not work for pexels image
Please check how it is handled on project thumbnail